INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:06:37] "POST / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:06:38] "GET /static/logo.png HTTP/1.1" 200 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:17:42] "POST / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:17:42] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
INFO:root:Received POST request
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:17:58] "GET /?authuser=1 HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:17:58] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:18:05] "[32mGET /google HTTP/1.1[0m" 302 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:18:05] "[32mGET /google_login/google HTTP/1.1[0m" 302 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/tzanouangeliki/flask_e2e_project/app.py", line 78, in index
    return render_template('index.html', bmi=bmi, error=error)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 152, in render_template
    return _render(app, template, context)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 133, in _render
    rv = template.render(context)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/tzanouangeliki/flask_e2e_project/templates/index.html", line 18, in top-level template code
    {% if google.authorized %}
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 485, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'google' is undefined
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:23:46] "[35m[1mGET /?authuser=1 HTTP/1.1[0m" 500 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/tzanouangeliki/flask_e2e_project/app.py", line 78, in index
    return render_template('index.html', bmi=bmi, error=error)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 151, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.9/dist-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:32:20] "[35m[1mGET /?authuser=1 HTTP/1.1[0m" 500 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/tzanouangeliki/flask_e2e_project/app.py", line 87, in index
    return render_template('index.html', bmi=bmi, error=error)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 151, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.9/dist-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:51:36] "[35m[1mGET /?authuser=1 HTTP/1.1[0m" 500 -
INFO:root:Received POST request
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/tzanouangeliki/flask_e2e_project/app.py", line 87, in index
    return render_template('index.html', bmi=bmi, error=error)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 151, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.9/dist-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:51:37] "[35m[1mGET /?authuser=1 HTTP/1.1[0m" 500 -
INFO:root:Received POST request
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/tzanouangeliki/flask_e2e_project/app.py", line 87, in index
    return render_template('index.html', bmi=bmi, error=error)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 151, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.9/dist-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:51:37] "[35m[1mGET /?authuser=1 HTTP/1.1[0m" 500 -
INFO:root:Received POST request
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/tzanouangeliki/flask_e2e_project/app.py", line 87, in index
    return render_template('index.html', bmi=bmi, error=error)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 151, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.9/dist-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.9/dist-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:51:38] "[35m[1mGET /?authuser=1 HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:51:45] "[33mGET /google HTTP/1.1[0m" 404 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:55:23] "[33mGET /google HTTP/1.1[0m" 404 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:55:24] "[33mGET /google HTTP/1.1[0m" 404 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:56:07] "[33mGET /google HTTP/1.1[0m" 404 -
INFO:root:Received POST request
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:56:14] "GET /?authuser=1 HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 02:56:14] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 03:11:35] "GET /?authuser=1 HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 03:11:35] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 03:23:47] "GET /?authuser=1 HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 03:23:47] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
INFO:root:DATABASE_URL from environment: mysql+mysqlconnector://ang_final:lilika123!@angfinal.mysql.database.azure.com:3306/angfinal
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:30:44] "GET /?authuser=1&redirectedPreviously=true HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:30:44] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:30:45] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
INFO:root:DATABASE_URL from environment: mysql+mysqlconnector://ang_final:lilika123!@angfinal.mysql.database.azure.com:3306/angfinal
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:47:20] "GET /?authuser=1 HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:47:20] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.88.0.3:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:root:Received POST request
INFO:root:DATABASE_URL from environment: mysql+mysqlconnector://ang_final:lilika123!@angfinal.mysql.database.azure.com:3306/angfinal
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:48:09] "GET /?authuser=1 HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:48:09] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
INFO:root:Received POST request
INFO:root:DATABASE_URL from environment: mysql+mysqlconnector://ang_final:lilika123!@angfinal.mysql.database.azure.com:3306/angfinal
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:48:34] "POST / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Dec/2023 19:48:34] "[36mGET /static/logo.png HTTP/1.1[0m" 304 -
